Form controls - Remove `@isInvalid` from `checkbox`, `radio` and `toggle` components

didoo opened the pull request:
### :pushpin: Summary

Now that the `checkbox`, `radio` and `toggle` components don't have anymore a visual "invalid" state (see #441) we can remove all the code related to the `@isInvalid` argument for these components.

### :hammer_and_wrench: Detailed description

In this PR I have:
- removed `@isInvalid` argument handling from `Checkbox/Radio/Toggle` components
- removed `@isInvalid` from documentation for `Checkbox/Radio/Toggle` components

***

### 👀 How to review

👉 Review commit-by-commit or by files changed

Reviewer's checklist:

- [ ] +1 Percy if applicable

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

alex-ju reviewed (commented):
I only reviewed the last two commits on this one (I think the other ones will go away at rebase) – looks good, but we need to update a few 'integration' tests that cover these cases.

didoo commented:
> I only reviewed the last two commits on this one (I think the other ones will go away at rebase) 

Not sure what _exactly_ happened. In any case I have rebased and force-pushed a new branch.

> but we need to update a few 'integration' tests that cover these cases.

Yes. You're right. Fixed now.

alex-ju reviewed (approved):
🙌
